runtime/doc/api.txt

```diff
@@ -1396,8 +1396,8 @@ nvim_set_hl({ns_id}, {name}, {*val})                           *nvim_set_hl()*
                  • link: name of another highlight group to link to, see
                    |:hi-link|.
                  • default: Don't override existing definition |:hi-default|
-                 • ctermfg: Sets foreground of cterm color |highlight-ctermfg|
-                 • ctermbg: Sets background of cterm color |highlight-ctermbg|
+                 • ctermfg: Sets foreground of cterm color |ctermfg|
+                 • ctermbg: Sets background of cterm color |ctermbg|
```

Comment:
Since "ctermbg", "guifg", etc. are common and unambiguous, I opted to remove the `highlight-` prefix.